torchaudio/models/conformer.py

```diff
@@ -22,16 +22,19 @@ class _ConvolutionModule(torch.nn.Module):
         input_dim (int): input dimension.
         num_channels (int): number of depthwise convolution layer input channels.
         depthwise_kernel_size (int): kernel size of depthwise convolution layer.
+        dropout (float, optional): dropout probability. (Default: 0.0)
         bias (bool, optional): indicates whether to add bias term to each convolution layer. (Default: ``False``)
+        use_group_norm (bool, optional): use GroupNorm rather than BatchNorm. (Default: False)
     """
 
     def __init__(
         self,
         input_dim: int,
         num_channels: int,
         depthwise_kernel_size: int,
-        bias: bool = False,
         dropout: float = 0.0,
+        bias: bool = False,
```

Comment:
Moving `bias` parameter is BC-breaking. Is this intended?

Reply:
sorry what does it mean by BC-breaking? I just wanted to gather bool-type inputs together, after int/float inputs

Reply:
Changing the order of function arguments breaks backward compatibility.

Say, a library, FooLibrary of v1 has a function like the following

```python
foo(a: int, b: str)
```

and in my project code I use it as

```python
result = foo(3, "foo")
```

Now, if the library code is changed like the following in v2 or something,

```python
foo(b: str, a: int)
```

Then simply upgrading my environment from FooLibrary v1 to v2, my project code stops working, unless I change my own code. The kind of changes that force client code change, is backward-incompatible, and it has to be communicated with users (deprecation). In PyTorch, we take at least two-release cycles for deprecating a public API.

Reply:
however the said class is not a public API, rather an internal one, so I guess it's kinda okay. Although I would say it's unnecessary if it's for the sake of cosmetics. @hwangjeff thoughts?

Reply:
it's an internal function so it should be ok to change, though I agree it's really a cosmetics change...